parsers/FO.py

```diff
@@ -44,48 +54,128 @@ def fetch_production(
             f"This parser cannot retrieve data for zone {zone_key!r}",
         )
 
-    if target_datetime is not None:
-        # There is a API endpoint at https://www.sev.fo/api/elproduction/last7days
-        # but it's currently returning nothing at all. Last checked on 2022-08-09
-        raise ParserException(
-            PARSER, "This parser is not yet able to parse past dates", zone_key=zone_key
-        )
+    now = datetime.now(timezone.utc)
+    target_datetime = (
+        now if target_datetime is None else target_datetime.astimezone(timezone.utc)
+    )
+
+    # API provides max one day of backlog
+    target_date_utc_to = datetime.combine(target_datetime, time(), tzinfo=timezone.utc)
+    target_date_utc_from = target_date_utc_to - timedelta(days=1)
+
+    # API takes local days as query parameters and returns data for the corresponding interval as UTC
+    target_date_local_to = target_date_utc_to.astimezone(tz=TIMEZONE)
+    target_date_local_from = target_date_utc_from.astimezone(tz=TIMEZONE)
+
+    # during SDT local time is UTC + 0 so day boundaries align
+    # but during DST local time is UTC + 1 so the data for UTC 23:00 is at 00:00 of the next local day, which will thus need to be requested
+    if target_date_local_to.dst():
+        target_date_local_to = target_date_local_to + timedelta(days=1)
```

Comment:
note: for example compare the alignment of the data returned on a SDT day 

https://www.sev.fo/api/elproduction/search?type=item&group=powerplant&from=20230215170000&to=20230216170000

vs a DST day

https://www.sev.fo/api/elproduction/search?type=item&group=powerplant&from=20230715170000&to=20230716170000